Description: Exclude everything except "jdk1.5/**" when building
 1.5 specific source code.
--- a/src/build.xml
+++ b/src/build.xml
@@ -72,6 +72,7 @@
   <target name="compile-jdk15" if="jdk15">
     <javac srcdir="src"
            destdir="${classes}"
+           excludes="org/**/*.java"
            includes="org/**/jdk15/*.java"
            deprecation="on"
            debug="${debug}"
